app/views/request/_sidebar.html.erb

```diff
@@ -31,6 +31,9 @@
                locals: { citations: citations, info_request: info_request } %>
   </div>
 
+  <%= render partial: 'request/share_by_link',
+             locals: { info_request: info_request } %>
```

Comment:
Need to remove this. Looks like dead9f2 got lost on rebase.

Reply:
Fixed with #5612